spec/paperclip/validators/attachment_size_validator_spec.rb

```diff
@@ -45,7 +46,7 @@ def self.should_not_allow_attachment_file_size(size, options = {})
 
       it "adds error to dummy object" do
         assert @dummy.errors[:avatar_file_size].present?,
-          "Unexpected error message on :avatar_file_size"
+          "Error not added to :avatar_file_size"     
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Trailing whitespace detected.